TKGame/Enemy.cs

```diff
@@ -63,18 +65,50 @@ public override void Update(GameTime gameTime)
 
         public void Move()
         {
-            Position.X += Velocity.X;                       // Enemy initially starts moving to right
+            // getter for player class
+            Player player = EntityManager.GetEntities().FirstOrDefault(x => x.entityName == "player" && x is Player) as Player;
 
-            if (Position.X == rightBoundary)
+            // check if player is null
+            if (player != null)
```

Comment:
We should probably have an else attached to this, the enemy is just going to stand still if the player is null for some reason